src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -72,9 +73,13 @@ public ObjectionService(ObjectionRepository objectionRepository,
     }
 
     @Override
-    public String createObjection(String requestId, String companyNumber, String ericUserId, String ericUserDetails) {
+    public ObjectionResponseDTO createObjection(String requestId, String companyNumber, String ericUserId, String ericUserDetails) {
         Map<String, Object> logMap = buildLogMap(companyNumber, null, null);
         logger.infoContext(requestId, "Creating objection", logMap);
+        ObjectionResponseDTO response = new ObjectionResponseDTO();
+
+        // TODO OBJ-231 process query result and return eligibility status
+        ObjectionStatus obStat = ObjectionStatus.OPEN;
```

Comment:
can you rename obStat to objectionStatus

Reply:
Renamed